### ISSUE TITLE
no sound on new week start

### ISSUE THREAD

**Issue by kambala-decapitator** [vcmi/vcmi#825]
When new week starts, respective dialog appears. Only after dismissing it the "new week" sound is played. IIRC in original game the sound is played in parallel to the dialog.

Use case: minimised the game during AI turn, was waiting for "new day" sound to switch back.

**Comment by Nordsoft91** [comment]
Unstable reproducing

**Comment by kambala-decapitator** [comment]
@IvanSavenko just discovered same issue with "player eliminated" dialog. Should I open a separate issue?

**Comment by IvanSavenko** [comment]
There is "player eliminated" sound?
Open a separate issue.
I assume issue is similar - sound does plays but at a wrong time?

**Comment by kambala-decapitator** [comment]
no, I mean the "player eliminated" dialog must be dismissed first, only after that "new week" sound is played. Will open issue.
